README.md

```diff
@@ -468,6 +469,158 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+If you need the full power of an SDK component, you can specificy a custom Direct Answers card.
+This component will be given the directAnswer data from the query, the same data the built-in component
+operates on. You can expect that object to look something like the below:
+
+```js
+{
+  type: "FIELD_VALUE",
+  answer: {
+    entityName: "Entity Name",
+    fieldName: "Phone Number",
+    fieldApiName: "mainPhone",
+    value: "+11234567890",
+    fieldType: "phone" 
+  },
+  relatedItem: { 
+    verticalConfigId: 'people',
+    data: { ... }
+  }
+}
+```
+
+An SDK component needs a corresponding template. This can be added either inline by changing the constructor to
```

Comment:
I would say "The new Direct Answers card component". The README states elsewhere that components, in general, need HBS templates. These new instructions reference "SDK component" and "component", but don't mention that the component they are creating is for a new Direct Answers card. I think that's important to explicitly include early on.